java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -346,14 +388,26 @@ function activateWithJDK(specifiedJDK: string | null, context: ExtensionContext,
             const yes = "Install GPLv2+CPEx code";
             window.showErrorMessage("Additional Java Support is needed", yes).then(reply => {
                 if (yes === reply) {
-                    let installProcess = launcher.launch(info, "--modules", "--install", ".*nbjavac.*");
+                    vscode.window.setStatusBarMessage("Preparing Apache NetBeans Language Server for additional installation", 2000);
+                    restartWithJDKLater = function() {
+                        log.appendLine("Ignoring request for restart of Apache NetBeans Language Server");
+                    };
+                    killNbProcess(false);
+                    let installProcess = launcher.launch(info, "-J-Dnetbeans.close=true", "--modules", "--install", ".*nbjavac.*");
                     let logData = function(d: any) {
                         log.append(d.toString());
                     };
                     installProcess.stdout.on('data', logData);
                     installProcess.stderr.on('data', logData);
                     installProcess.on('close', function(code: number) {
                         log.append("Additional Java Support installed with exit code " + code);
+                        // let the updater to settle down.
+                        activateWithJDK(specifiedJDK, context, log, notifyKill);
```

Comment:
...let the installer start new copy of Apache NetBeans Language Server explicitly.